packages/driver/src/cy/commands/actions/trigger.ts

```diff
@@ -117,12 +117,16 @@ export default (Commands, Cypress, cy, state, config) => {
         subject = options.$el.first()
       }
 
+      const subjectChain = cy.subjectChain()
```

Comment:
This is an example of the simplest possible case - `.trigger()` requires no additional logic. It reads the current subject chain and passes a function resolving it in to `$actionability.verify` as `subjectFn`.